.github/ISSUE_TEMPLATE/giskard_bug_template.yaml

```diff
@@ -0,0 +1,136 @@
+    attributes:
+      label: Issue Type
+      description: What type of issue would you like to report?
+      multiple: false
+      options:
+        - Bug
+        - Build/Install
+        - Performance
+        - Documentation Bug
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/Giskard-AI/giskard/blob/main/ISSUES.md),we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub.
+
+  # - type: markdown
+  #   attributes:
+      # value: | 
+      #   You can collect some of this information using our environment capture [script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)  You can also obtain the TensorFlow version with: <br> 1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` <br>2. TF 2.0: `python -c "import giskard ; print(tf.version.GIT_VERSION, tf.version.VERSION)"`
+
+  - type: dropdown
+    id: source
+    attributes:
+      label: Source
+      description: Giskard installed from
+      options:
+        - source
+        - binary
+    validations:
+      required: true
+  - type: input
+    id: giskardversion
+    attributes:
+      label: Giskard Version
+      description: 
+      placeholder: ex,. giskard 1.3
+    validations:
+      required: true
+  - type: dropdown
+    id: Code
+    attributes:
+      label: Custom Code
+      description: 
+      options:
+        - "Yes"
+        - "No"
+    validations:
+      required: true
+  - type: input
+    id: OS
+    attributes:
+      label: OS Platform and Distribution
+      description: 
+      placeholder: e.g., Linux Ubuntu 16.04
+    validations:
+      required: false
+  # - type: input
+  #   id: Mobile
+  #   attributes:
+  #     label: Mobile device
+  #     description: 
+  #     placeholder: e.g., Linux Ubuntu 16.04
+  #   validations:
+  #     required: false
+  - type: input
+    id: Python
+    attributes:
+      label: Python version
+      description: 
+      placeholder: e.g., 3.9
+    validations:
+      required: false
+  # - type: input
+  #   id: Bazel
+  #   attributes:
+  #     label: Bazel version
+  #     description: if compiling from source
+  #     placeholder: 
+  #   validations:
+  #     required: false
+  # - type: input
+  #   id: Compiler
+  #   attributes:
+  #     label: GCC/Compiler version
+  #     description: if compiling from source
+  #     placeholder: 
+  #   validations:
+  #     required: false
+  # - type: input
+  #   id: Cuda
+  #   attributes:
+  #     label: CUDA/cuDNN version
+  #     description:
+  #     placeholder: 
+  #   validations:
+  #     required: false
+  # - type: input
+  #   id: Gpu
+  #   attributes:
+  #     label: GPU model and memory
+  #     description: if compiling from source
+  #     placeholder: 
+  #   validations:
+  #     required: false
```

Comment:
Comment should be deleted

Reply:
As mentioned in the weekly, comments were intentionally kept for @andreybavt to check if some of them can be useful for us.

Reply:
I'd only leave a section that asks to submit currently installed python packages in the environment, for example by running `pip list --format freeze`